BankManager.swift

```diff
@@ -5,3 +5,75 @@
 //
 
 import Foundation
+
+class Bank {
+    let teller = Teller(tellerIndex: 1)
+    var client = Queue<Int>()
+    
+    func startService() {
+        showInitialMenu()
+        
+        guard let input = receiveUserInput() else { return }
+        switch input {
+                case SelectMenu.start.rawValue:
+                    enqueueClient(pickRandomClientsNumber())
+                    conductService(by: teller)
+                    startService()
+                case SelectMenu.end.rawValue:
+                    return
+                default:
+                    print(Messages.unknownInputMessage.rawValue)
+                    startService()
+                }
+    }
+    
+    private func showInitialMenu() {
+        print(Messages.initialChoiceMessage.rawValue)
+        print("입력", terminator: " : ")
+    }
+    
+    private func receiveUserInput() -> String? {
+        let userInput = readLine()
+        
+        guard let input = userInput else {
+            return nil
+        }
+        
+        return input
+    }
+    
+    private func pickRandomClientsNumber() -> Int {
+        let randomClientsNumber = Int.random(in: 10...30)
+        return randomClientsNumber
+    }
+    
+    private func enqueueClient(_ clientNumber: Int) {
+        for index in 1...clientNumber {
+            client.enqueue(index)
+        }
+    }
+    
+    private func conductService(by teller: Teller) {
+        var workhours: Double = 0
+        var clientsCount: Int = 0
+        
+        while !client.isEmpty {
+            guard let firstClientNumber = client.front else {
+                return
+            }
```

Comment:
guard문은 너무 길지 않으면 한줄로 쓰는게 가독성이 더 좋습니다 😀
`client.front`면 클라이언트가 옵셔널 처리가 되어서 나올 것 같은데, clientNumber가 나와서 어색하네요! 개선해보면 좋을 것 같아요!
```suggestion
            guard let firstClientNumber = client.front else { return }
```

Reply:
한줄로 수정하였고 theFirstClient 로 수정하였습니다!